htdocs/accountancy/bookkeeping/card.php

```diff
@@ -651,8 +651,8 @@
 					print "</tr>\n";
 				}
 
-				$total_debit = price2num($total_debit);
-				$total_credit = price2num($total_credit);
+				$total_debit = price2num($total_debit, 2);
```

Comment:
Second parameter should be 'MT' (some countries use more or less than 2 digits for their currency)

Reply:
@eldy i just did it